components/amara/sources/common.mjs

```diff
@@ -0,0 +1,129 @@
+import amara from "../amara.app.mjs";
+import constants from "../constants.mjs";
+
+export default {
+  props: {
+    amara,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      label: "Polling schedule",
+      description: "How often to poll the Amara API",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+    team: {
+      description: "The team slug of the team to watch. Passing in `null` or empty will return only resources that are in the public area.",
```

Comment:
Should the second sentence from this description be removed now that this prop has `optional: false`?